LibraBFT/ImplShared/Consensus/Types.agda

```diff
@@ -63,11 +63,16 @@ module LibraBFT.ImplShared.Consensus.Types where
              (rsHighestCommittedRound ∷ rsCurrentRound ∷ rsPendingVotes ∷
               rsVoteSent ∷ [])
 
+  record ObmNeedFetch : Set where
```

Comment:
I guess this is just scaffolding while we decide how to handle modeling this?  I suggest a comment to that effect, or a `TODO` if we plan to elaborate this more.

Reply:
I plan to put more extensive comments in `LibraBFT.Impl.IO.OBM.ObmNeedFetch`
because that is where the magic would happen.